train/output/metrics_LaBSE.md

```
@@ -0,0 +1,264 @@
+| Throughput | 24633 sps |
+
+## Overall evaluation
+
+| Metric | Value |
+|---|---|
+| Accuracy | 0.9603 |
+| Weighted F1 | 0.9596 |
+
+### Classification report
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)